docs/en/drivers/android-uiautomator2.md

```diff
@@ -12,11 +12,11 @@ repo.
 
 Older Android-based drivers include:
 * The [UiAutomator Driver](android-uiautomator.md)
```

Comment:
Elsewhere links like this are the full path from `/docs` (e.g., line 14 of the espresso driver page).